testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/BruteForceTest.java

```diff
@@ -362,17 +362,6 @@ public void testEmail() throws Exception {
         clearAllUserFailures();
     }
 
```

Comment:
@k-tamura out of curiosity why the test below was removed?

Reply:
Sorry. I didn't notice build failure for my commit. I have removed _BruteForceTest.testBrowserMissingPassword()_ and confirmed build success on my local. I think this method is not needed as I previously commented:

> The aim of account lock is to avoid brute force attacks. I think brute force detection does not need to work for login without password because malicious users never attack without password.

Reply:
> Sorry. I didn't notice build failure for my previous commit. I have removed _BruteForceTest.testBrowserMissingPassword()_ and confirmed build success on my local.

Absolutely no worries. But I'm not so sure about removing the test, blank password or missing password it also count as password. Of course, that would be the consequence of a very poor password policy. But can happen and does not hurt to keep it.

Reply:
Do you think the method should be left as follows:

```java
    @Test
    public void testBrowserMissingPassword() throws Exception {
        loginSuccess();
        loginMissingPassword();
        loginMissingPassword();
        loginSuccess();
    }
```